modules/preferred-installation-requirements-ibm-z.adoc

```diff
@@ -59,6 +61,6 @@ When installing in a z/VM environment, you can also bridge HiperSockets with one
 |One machine
 
 |===
-1.To ensure the availability of integral components in an overcommitted environment, increase the priority of the control plane by using the CP command `SET SHARE`. Do the same for infrastructure nodes, if they exist. See link:https://www.ibm.com/docs/en/zvm/latest?topic=commands-set-share[SET SHARE] in {ibm-name} Documentation.
+1. To ensure the availability of integral components in an overcommitted environment, increase the priority of the control plane by using the CP command `SET SHARE`. Do the same for infrastructure nodes, if they exist. See link:https://www.ibm.com/docs/en/zvm/latest?topic=commands-set-share[SET SHARE] in {ibm-name} Documentation.
```

Comment:
```suggestion
1. To ensure the availability of integral components in an overcommitted environment, increase the priority of the control plane by using the CP command `SET SHARE`. Do the same for infrastructure nodes, if they exist. See link:https://www.ibm.com/docs/en/zvm/latest?topic=commands-set-share[SET SHARE] ({ibm-name} Documentation).
```
There was a style change for external docs introducing parens for the docset name. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute